tests/approaches/test_oracle_approach.py

```diff
@@ -603,27 +603,57 @@ def test_oracle_approach_repeated_nextto_painting():
     """Tests for OracleApproach class with RepeatedNextToPaintingEnv."""
     utils.reset_config({
         "env": "repeated_nextto_painting",
-        "num_train_tasks": 1,
         "num_test_tasks": 1,
+        "painting_num_objs_test": [1],
+        # NOTE: We found hff to make planning for this problem significantly
+        # faster, and thus use it here so that the tests run quickly!
         "sesame_task_planning_heuristic": "hff"
     })
-    # NOTE: We found hff to make planning for this problem significantly
-    # faster, and thus use it here so that the tests run quickly!
     env = RepeatedNextToPaintingEnv()
-    env.seed(3210)  # This random seed necessary to get full coverage.
+    env.seed(123)
```

Comment:
this should be unnecessary right? seed is called in initialization, and 123 is the default

Reply:
opened an issue #600 i can do this later